### DISCUSSION TITLE
Where should I begin to learn R for Outbreak analytics?

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/epiverse-trace.github.io#91] [General]
Hi all,

This is to share that I started a [draft PR](https://github.com/epiverse-trace/epiverse-trace.github.io/pull/90) for the website with a curated list of existing learning resources in the field.

Users interested in using the Epiverse-TRACE packages should be able to self-assess and choose the material of their preference according to the self-perceived gaps, given specific learning goals at different stages.

Note: I decided to upload this as a draft PR to facilitate in-line comments or suggestions in the "Files changed" tab.

I'm interested to read your questions or suggestions on the topic.

**Comment by Bisaloo** [comment]
https://skillai.io/ can generate learning paths for any skills and can serve as a starting point for this reflection. Example of the output for "R for outbreak analytics":

![roadmap(1)](https://github.com/epiverse-trace/epiverse-trace.github.io/assets/10783929/dca8dea0-9027-46cb-a1a6-9ca198747225)

**Reply by avallecam** [reply]
That is really nice, Hugo. I'll add it at the beginning of the post and reuse some of these titles to find resources.

**Comment by avallecam** [comment]
I want to share one of the questions from the PR here.

> - Should this be a repository instead? This list of resources can have a periodic review process over time.

My answer now is yes. This will provide much flexibility for updates. The plan would be to translate the content from the draft PR to a new repository. Then, use this space as an announcement space for the new repository. Additional implications would be that related contributions, discussions and issues/pr for updates could be handled separately from the website repo.

I am open to reading more thoughts about this.
